bin/truffleruby

```diff
@@ -93,7 +93,8 @@ if [ $on_graalvm = false ]; then
 
     if [ "$mx_time" -gt "$mvn_time" ]; then
         binary_truffle="$root/mx.imports/binary/truffle/mxbuild"
-        source_truffle="$(dirname "$root")/truffle/mxbuild"
+        source_truffle_suite_py=("$(dirname "$root")"/*/*/"mx.truffle/suite.py")
```

Comment:
This seems very dangerous, it could pick a `mx.truffle/suite.py` in any sibling directory of our repo.
Just to get an idea, I have 12 results for this currently.
Also, this search takes forever the first time it's performed uncached in my `~/code/truffleruby` where I have many code repositiories:
```
$ time zsh -c 'echo */*/mx.truffle/suite.py'
graal-core/truffle/mx.truffle/suite.py ...
zsh -c 'echo */*/mx.truffle/suite.py'  0.06s user 0.25s system 5% cpu 5.626 total
```
So, I believe we need to be more precise.
Maybe just `graal/truffle/mx.truffle/suite.py` ?